src/styles/App.css

```diff
@@ -0,0 +1,52 @@
+body {
+background-color: #282c34;
+}
+
+.App {
+  text-align: center;  
+  color: beige;
+}
+
+header {
+  /* color: azure; */
```

Comment:
Not sure if this is still being used, but if not then the commented code can be removed